tests/results/dp-perf/edge/edge-oss.md

```diff
@@ -7,45 +7,45 @@ NGINX Plus: false
 GKE Cluster:
 
 - Node count: 12
-- k8s version: v1.29.4-gke.1043004
+- k8s version: v1.29.5-gke.1091002
 - vCPUs per node: 16
 - RAM per node: 65855012Ki
 - Max pods per node: 110
-- Zone: us-east1-b
+- Zone: us-west1-b
```

Comment:
Why is the zone different @lucacome?

Reply:
I decided to move the tests to a low carbon impact [location](https://cloud.google.com/sustainability/region-carbon#data)

Reply:
Ah, that's right